splink/comparison_level_library.py

```diff
@@ -161,12 +168,13 @@ def __init__(
                     "transforms applied to it such as lower(), "
                     "substr() etc."
                 )
-
-            config["tf_adjustment_column"] = col_name
-            config["tf_adjustment_weight"] = 1.0
             # leave tf_minimum_u_value as None
```

Comment:
```suggestion
            # leave tf_minimum_u_value as None
            # Since we know that it's a pure column reference it's fine to assign the
            #  raw unescaped value to the dict - it will be processed via `InputColumn` when
            # the dict is read
            
```